update_automation_example.py

```diff
@@ -0,0 +1,122 @@
+                        help='automation script name(.py or .js)',
+                        required=False,
+                        default=sys.stdin)
+    options = parser.parse_args()
+    if options.name is None:
+        if options.file.name == '<stdin>':
+            if options.test:
+                options.name = "test"
+            else:
+                raise RuntimeError(
+                    'Error parsing options - name is required')
+        else:
+            options.name = os.path.splitext(
+                os.path.basename(options.file.name))[0]
+    return options
+
+
+class TestAutomation(unittest.TestCase):
+    def setUp(self):
+        automation = {
+            "name": self.name,
+            "script": "demisto.results('Hello, world!')"
+        }
+        automations = self.client.SaveAutomation(automation)
+        self.id = automations["scripts"][0]["id"]
+
+    def tearDown(self):
+        automation = {
+            "name": self.name,
+            "id": self.id
+        }
+        self.client.DeleteAutomation(automation)
+        self.id = None
+
+    def test_search(self):
+        automations = self.client.SearchAutomation("name:{}".format(self.name))
+        self.assertTrue("scripts" in automations)
+        self.assertTrue(len(automations["scripts"]) == 1)
+        self.assertTrue("id" in automations["scripts"][0])
+
+    def test_delete(self):
+        automation = {
+            "name": self.name,
+            "id": self.id
+        }
+        client.DeleteAutomation(automation)
+        automations = self.client.SearchAutomation("name:{}".format(self.name))
+        self.assertTrue("scripts" in automations)
+        self.assertTrue(len(automations["scripts"]) == 0)
+
+    def test_save_new(self):
+        automation = {
+            "name": self.name,
+            "id": self.id
+        }
+        client.DeleteAutomation(automation)
+        automation = {
+            "name": self.name,
+            "script": "demisto.results('Hello, world!')"
+        }
+        automations = self.client.SaveAutomation(automation)
+        self.assertTrue("scripts" in automations)
+        self.assertTrue(len(automations["scripts"]) == 1)
+        self.assertTrue("id" in automations["scripts"][0])
+        self.id = automations["scripts"][0]["id"]
+
+    def test_load(self):
+        automation = self.client.LoadAutomation(self.id)
+        self.assertTrue("id" in automation)
+        self.assertTrue(automation["id"] == self.id)
+
+    def test_update(self):
+        script = "demisto.results('Hello, world!!!!')"
+        automations = self.client.UpdateAutomation(
+            None, self.name, script=script)
+        self.assertTrue("scripts" in automations)
+        self.assertTrue(len(automations["scripts"]) == 1)
+        self.assertTrue("version" in automations["scripts"][0])
+        self.assertTrue(automations["scripts"][0]["version"] == 2)
```

Comment:
```python
self.assertEqual(len(automations["scripts"]), 1)
self.assertEqual(automations["scripts"][0]["version"], 2)
```

Reply:
use assertEqual instead of assertTrue(ea1a450).